src/components/tables/Collections/Rows.tsx

```diff
@@ -34,29 +40,51 @@ export const tableColumns = [
     },
 ];
 
-function Rows({ data }: Props) {
```

Comment:
Make this file match the other two tables where `rows` and `row` are broken apart.